src/index.js

```diff
@@ -0,0 +1,113 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { Modal } from 'bootstrap';
+import './styles.css';
+import Storage from './storage';
+import Project from './projects';
+import UI from './ui';
+import Task from './task';
+
+document.querySelector('#add-task').addEventListener('click', UI.setMinDate());
+
+const loadAll = () => {
+  UI.loadProjects();
+  UI.loadTodayTasks();
+};
+
+const modalEl = document.getElementById('exampleModal');
+
+const closeModal = () => {
+  const modal = Modal.getInstance(modalEl);
+  modal.hide();
+  modalEl.addEventListener('hidden.bs.modal', () => {
+    modal.dispose();
+  }, { once: true });
+};
+
+window.addEventListener('DOMContentLoaded', loadAll);
+
+document.querySelector('#title').addEventListener('keyup', () => {
+  const error = document.getElementById('taskError');
+  error.textContent = '';
+});
+
+document.querySelector('#new_project').addEventListener('keyup', () => {
+  const error = document.getElementById('projectError');
+  error.textContent = '';
+});
+document.querySelector('#project-form').addEventListener('submit', (e) => {
+  e.preventDefault();
+  const proName = document.querySelector('#new_project').value;
+  const xyz = proName.toUpperCase();
+  const error = document.getElementById('projectError');
+
+  if (Storage.getProject(xyz) !== undefined) {
+  // Changing content and color of content
+    error.textContent = 'A Project with that name already exists';
+    error.style.color = 'red';
+    return;
+  }
+
+  const abc = new Project(xyz);
+  const todo = Storage.getTodoList();
+  todo.addProject(abc);
+  Storage.saveTodoList(todo);
+  UI.clearProjectList();
+  UI.loadProjects();
+  UI.loadTask(xyz);
+  // UI.clearProjectForm();
+});
+
+document.querySelectorAll('.named-project-list').forEach((project) => {
+  project.addEventListener('click', (e) => {
+    UI.loadTask(e.target.innerText);
+  });
+});
+
+document.querySelector('#task-form').addEventListener('submit', (e) => {
+  e.preventDefault();
+  const previousTitle = document.querySelector('#title').placeholder;
+
+  // var taskForm = new Modal(document.getElementById('exampleModal'), {
```

Comment:
Remove commented code

Reply:
Hi Forison,

Thanks for the feedback on my code review request.

The way I set up the project, It is supposed to load all of Today's tasks on load. So, "Today's task" is supposed to be like the default project. 

I will make changes as regards the commented code as well.